sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -152,7 +152,10 @@ class DateTime final : public _detail::Clock::time_point {
     AllDigits,
 
     /// Drop all the fractional time digits.
-    Truncate
+    Truncate,
+
+    /// Include only the date portion, skipping time and time zone.
+    Date,
   };
 
   /**
```

Comment:
The comment for `Rfc3339` should express which construct in `Rfc3339` the DateFormat conforms to (I believe it's `date-time`, based on the code, but the comment should be specific).